akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Users2x.scala

```diff
@@ -118,6 +118,16 @@ object Users2x {
     }
   }
 
+  def setUserName(users: Users2x, intId: String, newUserName: String): Option[UserState] = {
+    for {
+      u <- findWithIntId(users, intId)
+    } yield {
+      val newUser = u.modify(_.name).setTo(newUserName)

```

Comment:
Perhaps some validation would be necessary here - do not allow to change to empty string, to anything else we don't allow in bbb-web...

Reply:
There is definitely some validation missing. Do you recommend it to be done in akka-apps, html5 or both? Maybe also maxlength?

Reply:
I'll have to think about this some more. There's also the question of what do we do with bbb-web... For example if we don't touch it, API responses will contain the old name if I understand this correctly... Let's see if anyone else comments while I am making up my mind about this...

Reply:
Can confirm, API is still returning old name. I can try to propagate the user name change to bbb-web as well, if this is what we want. Waiting for some more thoughts on this...